src/plugin/arc/node/arcservicenode.js

```diff
@@ -185,17 +186,16 @@ class ArcServiceNode extends LoadingNode {
    */
   addImageLayer_(json) {
     const id = this.server_.getId() + BaseProvider.ID_DELIMITER + /** @type {string} */ (json['name']);
-    const extent = /** @type {Object<string, number>} */ (json['extent']);
-    const wkid = /** @type {number} */ (extent['spatialReference']['latestWkid']);
+    const extent = arc.readEsriExtent(/** @type {Object} */ (json['extent']), EPSG4326);
```

Comment:
Do we always intend to read the extent into 4326, or should we be using the current map projection? Does that get transformed later down the line?

Reply:
This is called when the server is loaded, and the projection may change by the time the layer is loaded. I decided to always read the projection as 4326, and `ArcImageLayerConfig` will handle transforming it to the current projection. This is similar to the approach in `AbstractTileLayerConfig`, which assumes 4326 unless specified in `extentProjection`.